landoui/pages.py

```diff
@@ -47,12 +48,13 @@ def signout():
 @pages.route('/logout')
 @oidc.oidc_logout
 def logout():
-    return_url = "{protocol}://{host}/signout".format(
-        protocol=os.environ['PROTOCOL'],
-        host=current_app.config['SERVER_NAME']
+    protocol = 'http' if os.environ['RUN_DEV_SERVER'] else 'https'
```

Comment:
Oops, I missed this the first time.  I prefer the setting of application internal variables from external configuration to happen all in the same place, and for Flask apps that's the `create_app()` factory method.  Can we read the env var in `create_app()`, figure out the protocol we are serving in `create_app()`, set a config value like `app.config['PROTOCOL']`, and read the new `app.config` var here?